WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -183,6 +185,7 @@
     @Inject AppConfig mAppConfig;
     @Inject ImageEditorFileUtils mImageEditorFileUtils;
     @Inject ExPlat mExPlat;
+    @Inject Set<Experiment> mExperiments;
```

Comment:
I'm wondering whether it would make sense to inject this directly in the `ExPlat` class.

Reply:
That would make sense, for sure. The reason I avoided doing that in the first place was because I didn't want to generate a circular dependency (`ExPlat` depends on `Experiment` that depends on `ExPlat`).

That said, I tried using `dagger.Lazy` to inject the `Set<Experiment>` in the `ExPlat` constructor and it actually ended up working pretty well and the code got cleaner, specially because we no longer need the `ExPlat::start` method. 

You can check the changes here: 3eb27b7 – thanks for the suggestion!